ethcore/src/miner/miner.rs

```diff
@@ -1429,15 +1438,22 @@ impl miner::MinerService for Miner {
 						service_transaction_checker.as_ref(),
 					);
 					queue.cull(client);
+					if is_internal_import {
```

Comment:
For my education, what is an "internal" import in this context?

Reply:
I think here it's a block that has been created after having sent transactions to your node, so triggered by a local RPC call rather than external transactions through the DevP2P network.

Reply:
Should it be named `is_local_import` you think?

Reply:
Actually it's for import blocks that have been sealed internally, sorry. So I think we should keep it that way.